### PR TITLE
post draft

### PR DESCRIPTION
**Pull request by maelle**
Goals of the post: discussing the topic of getting help with R package development, mention how we use questions as docs fodder, and feature an analysis of R-package-devel (hopefully at least a bit interesting).

Notes of improvements to be made

* Add alt text to pictures

* When mentioning topics in text after the topic words plot, add words in text because not everyone can see the picture with words by topic on it.

* Rm typos :smile_cat: 

* Make EDA explicit the 1st time it appears

### CODE REVIEW

File: content/post/2019-04-11-r-pkg-devel.md [hunk 1]
```diff
@@ -0,0 +1,681 @@
...
+output:
+  md_document:
+    variant: markdown_github
+    preserve_yaml: true
+---
+
+No matter how good your docs reading and search engine querying skills
+are, sometimes as an R package developer you’ll need to ask questions to
+your peers. Where to find them? R-hub has its own
+[feedback](https://github.com/r-hub/rhub) and
```
Comment (gaborcsardi) [inline content/post/2019-04-11-r-pkg-devel.md]: Link to `/issues` here?

File: content/post/2019-04-11-r-pkg-devel.md [hunk 2]
```diff
@@ -0,0 +1,681 @@
...
+[discussion](https://gitter.im/r-hub/community) venues, but what about R
+package development in general? In this blog post, we’ll have a look at
+the oldest channel for R package development help, the [R-package-devel
```
Comment (gaborcsardi) [inline content/post/2019-04-11-r-pkg-devel.md]: Well, not really the oldest, there was R-devel, and StackOverflow, etc. before.

File: content/post/2019-04-11-r-pkg-devel.md [hunk 3]
```diff
@@ -0,0 +1,681 @@
...
+    ##  4 murdoch@dunc@n @ending from gm@il@com (Duncan Murdoch)          68
+    ##  5 h.wickham at gmail.com (Hadley Wickham)                         60
+    ##  6 ligge@ @ending from @t@ti@tik@tu-dortmund@de (Uwe Ligges)       46
+    ##  7 edd @ending from debi@n@org (Dirk Eddelbuettel)                 45
+    ##  8 glennmschultz at me.com (Glenn Schultz)                         43
+    ##  9 csardi.gabor at gmail.com (=?UTF-8?B?R8OhYm9yIENzw6FyZGk=?=)    42
+    ## 10 i.ucar86 at gmail.com (=?UTF-8?B?ScOxYWtpIMOaY2Fy?=)            42
+    ## # … with 660 more rows
+
+*:wave: Gábor Csárdi, R-hub creator!*
```
Comment (gaborcsardi) [inline content/post/2019-04-11-r-pkg-devel.md]: Can we please remove this? :)

File: content/post/2019-04-11-r-pkg-devel.md (hunk 2 above)
Comment (maelle) [reply content/post/2019-04-11-r-pkg-devel.md]: oldest _specific_ then?

Comment (gaborcsardi) [reply content/post/2019-04-11-r-pkg-devel.md]: ok

File: content/post/2019-04-11-r-pkg-devel.md (hunk 3 above)
Comment (maelle) [reply content/post/2019-04-11-r-pkg-devel.md]: Ok, will stop the waving :grin:

**Comment by maelle** [comment]
Thanks, I'll merge and publish+publicize on Thursday!

**Pull request merged**